operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -201,7 +201,7 @@ mimir:
         compactor:
           url: http://{{ template "mimir.fullname" . }}-compactor.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
         distributor:
-          url: http://{{ template "mimir.fullname" . }}-distributor.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+          url: dns:///{{ template "mimir.fullname" . }}-distributor-headless.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.serverGrpcListenPort" . }}
```

Comment:
Need to test this, that serverGrpcListenPort is very suspicious

Reply:
ok, this works and TIL : https://grafana.com/docs/enterprise-metrics/v1.6.x/gateway/#client-side-load-balancing